zhaquirks/philips/rdm004.py

```diff
@@ -0,0 +1,197 @@
+"""Signify RDM004 device."""
+import logging
+from typing import Any, List, Optional, Union
+
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+import zigpy.types as t
+from zigpy.zcl import foundation
+from zigpy.zcl.clusters.general import (
+    Basic,
+    Groups,
+    Identify,
+    LevelControl,
+    OnOff,
+    Ota,
+    PowerConfiguration,
+)
+
+from zhaquirks.const import (
+    ARGS,
+    BUTTON,
+    COMMAND,
+    COMMAND_ID,
+    DEVICE_TYPE,
+    DOUBLE_PRESS,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    LONG_PRESS,
+    LONG_RELEASE,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PRESS_TYPE,
+    PROFILE_ID,
+    QUADRUPLE_PRESS,
+    QUINTUPLE_PRESS,
+    RIGHT,
+    SHORT_PRESS,
+    SHORT_RELEASE,
+    TRIPLE_PRESS,
+    TURN_ON,
+    ZHA_SEND_EVENT,
+)
+from zhaquirks.philips import PHILIPS, SIGNIFY
+
+DEVICE_SPECIFIC_UNKNOWN = 64512
+_LOGGER = logging.getLogger(__name__)
+
+
+class PhilipsBasicCluster(CustomCluster, Basic):
+    """Philips Basic cluster."""
+
+    attributes = Basic.attributes.copy()
+    attributes.update(
+        {
+            0x0031: ("philips", t.bitmap16, True),
+            0x0034: ("mode", t.enum8, True),    
```

Comment:
Some spaces is after the comma that the tests dont like.
Put  in `            0x0034: ("mode", t.enum8, True),` and it shall working OK